### PR TITLE
chore: add node 18

### PR DESCRIPTION
**Pull request by harish-sethuraman**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case
Add node 18 to test workflow

### CODE REVIEW

**Review by amareshsm — COMMENTED** [review]
remove node v17 it is not LTS

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [12.x, 14.x, 16.x, 17.x, 18.x]
```
Comment (harish-sethuraman) [inline .github/workflows/nodejs.yml:56]:
```suggestion
        node-version: [12.x, 14.x, 16.x, 18.x]
```

**Review by amareshsm — APPROVED** [review]

**Comment by alexander-akait** [comment]
Please update snapshots

**Comment by harish-sethuraman** [comment]
Closing in favour of https://github.com/webpack-contrib/less-loader/pull/463

**Pull request closed without merging**